kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateQuizSection.vue

```diff
@@ -185,57 +185,34 @@
         </KGrid>
 
         <AccordionContainer
-          :items="quizForge.activeQuestions.value"
-          @toggled="items => expandedQuestionIds = items"
+          :items="quizForge.activeQuestions.value.map(i => ({
```

Comment:
I would suggest that we can make `items` a generic array of objects that contains, for example, the `id` attribute (instead of something specific like "question_id"), and that from this we have the criteria to know which items to choose expand when the "expand all" button is clicked.

Reply:
Excellent idea!